add include_namespace (#773)
logzio/logz-docs: _source/logzio_collections/_log-sources/kubernetes.md

| flush_interval <span class="default-param">`5s`</span> | Interval, in seconds, to wait before invoking the next buffer flush. |
| max_retry_wait <span class="default-param">`30s`</span> | Maximum interval, in seconds, to wait between retries. |
| num_threads <span class="default-param">`2`</span> | Number of threads to flush the buffer. |

> | INCLUDE_NAMESPACE | <span class="default-param">`""`(All namespaces)</span> | Use if you wish to send logs from specific k8s namespaces, space delimited. Should be in the following format: <br> `kubernetes.var.log.containers.**_<<NAMESPACE-TO-INCLUDE>>_** kubernetes.var.log.containers.**_<<ANOTHER-NAMESPACE>>_**`. |

Reviewer: ```suggestion
| INCLUDE_NAMESPACE | <span class="default-param">`""`(All namespaces)</span> | Sends logs from all namespaces by default. To send logs from specific k8s namespaces, specify them in the following format, space delimited: <br> `kubernetes.var.log.containers.**_<<NAMESPACE-TO-INCLUDE>>_** kubernetes.var.log.containers.**_<<ANOTHER-NAMESPACE>>_**`. |
```
